src/reducers/errors.js

```diff
@@ -23,7 +23,7 @@ const reducer = (state = defaultState, action) => {
         return state;
       }
       return {
-        errors: [action.error, ...state.errors]
+        errors: [{ message: action.errorMessage }, ...state.errors]
```

Comment:
@craicoverflow  This should be investigated, since the change was there to fix an incongruence into the code.
If you look at line `21` it expects the `err.message` to be equal `action.errorMessage`, but then at line 26 (old code) it inserts the message into `errors` as a plain string.